code/unit_tests/organ_tests.dm

```diff
@@ -251,4 +251,141 @@
 	else
 		pass("All organs were removed and replaced correctly.")
 
-	return 1
\ No newline at end of file
+	return 1
+
+// ==============================================================================
+// Stumps shall not drop
+// ==============================================================================
+/datum/unit_test/stumps_shall_not_drop
+	name = "ORGAN: Stumps Shall Not Drop From a Gibbed Mob or Severed Limbs."
+
+/datum/unit_test/stumps_shall_not_drop/proc/find_stumps()
+	//Look for stumps that aren't deleted
+	var/list/found_stumps = (locate(/obj/item/organ/external/stump) in world)
```

Comment:
I don't think this works; you'd need to use a for loop if you want a list of stumps. As it stands, the body of the for loop will never run. However, since you check LAZYLEN, LAZYLEN of a single datum is always 0; therefore, the test will always pass even if stumps drop.
```suggestion
```

Reply:
You're right! Not sure why I thought it returned a list..